src/components/ebay-signal/index.marko

```diff
@@ -0,0 +1,5 @@
+$ var type = input.type;
+
+<span class=`signal signal--${input.type}`>
```

Comment:
Pass in the `...input`

Also do we need any `role` or aria attributes here? At least `role` would probably be needed.

Reply:
Good point. I'll have a think if we need anything and add it to the Skin docs if so.